src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,100 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static User user;
+    private static RegistrationService registrationService;
+    private static final long ID = 0;
+    private static final String LOGIN_VALID = "aboba@gmail.com";
+    private static final String PASSWORD_VALID = "123456";
+    private static final String LOGIN_INVALID = "aboba";
+    private static final String PASSWORD_INVALID = "12345";
+
+    private static final int AGE_VALID = 18;
+    private static final int AGE_INVALID = 17;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setId(ID);
+        user.setAge(AGE_VALID);
+        user.setLogin(LOGIN_VALID);
+        user.setPassword(PASSWORD_VALID);
+    }
+
+    @Test
+    void register_nullUser_notOk() {
+        assertThrows(RegistrationException.class,
+                () -> registrationService.register(null),
+                "User shouldn't be null");
```

Comment:
If I'm not mistaken this 3-rd parameter (message) is used when an exception is not raised. Here are some docs:
<img width="818" alt="image" src="https://github.com/mate-academy/jv-registration-validation/assets/53354844/3653cd98-c349-4d19-9fe5-544c4d7657c1">

So, it's better to provide something like 'RegistrationException error was expected' instead